### ISSUE TITLE
Allow Players to have a LONGER Player Tag

### ISSUE THREAD

**Issue by Blaze-Corderz** [efroemling/ballistica#737]
## Description
Let players have a player tag longer than 10 characters.

**Comment by MatiasMGz** [comment]
I believe that 12 characters (the limit that most websites use) is a perfect limit for account names!

**Comment by Blaze-Corderz** [comment]
Yeah 12 Charecters would be good enough.

**Comment by MatiasMGz** [comment]
It shouldn't be that difficult for efro to allow 12 characters names.

**Comment by Blaze-Corderz** [comment]
yes it shouldn't, idk if he will do it when he starts working on 2.0

**Comment by MatiasMGz** [comment]
2.0 is launching ports for other platforms. What you are referring to is 1.9!

**Comment by Blaze-Corderz** [comment]
oh, well 1.9 it is then

**Comment by MatiasMGz** [comment]
I am in favor of this, i wanted matias26157 as my name but it wasn't 9 characters so i couldn't get it.
